code/game/machinery/porta_turret/portable_turret.dm

```diff
@@ -448,8 +448,8 @@ DEFINE_BITFIELD(turret_flags, list(
 
 			if(iscyborg(sillycone))
 				var/mob/living/silicon/robot/sillyconerobot = A
-				if(LAZYLEN(faction) && (ROLE_SYNDICATE in faction) && sillyconerobot.emagged == TRUE)
-					continue
+				if((ROLE_SYNDICATE in faction) && !sillyconerobot.emagged && LAZYLEN(faction))
+					targets+= sillyconerobot
```

Comment:
The feature is good, but take a look at the `if(issilicon(A))` check.
This code is confusing, somewhat duplicated and I'm not entirely sure it does what you intend it to.
Have you tested this?

Reply:
> The feature is good, but take a look at the `if(issilicon(A))` check. This code is confusing, somewhat duplicated and I'm not entirely sure it does what you intend it to. Have you tested this?

yep its all tested and WORKS as intended. Only cyborgs that arent emagged or arent part of the synditeam will be shot down